docs/configure_swagger.md

```diff
@@ -1,24 +1,22 @@
 # View and test Harbor REST API via Swagger
-A Swagger file is provided for viewing and testing Harbor REST API. First, you should get the source code of Harbor:
-```sh
-   git clone git@github.com:vmware/harbor.git
-```
+A Swagger file is provided for viewing and testing Harbor REST API.
+
 ### Viewing Harbor REST API
 * Open the file **swagger.yaml** under the _docs_ directory in Harbor project;
 * Paste all its content into the online Swagger Editor at http://editor.swagger.io. The descriptions of Harbor API will be shown on the right pane of the page.
 
 ![Swagger Editor](img/swaggerEditor.png)
 
 ### Testing Harbor REST API
-From time to time, you may need to mannually test Harbor REST API. You can deploy the Swagger file into Harbor's service node. 
+From time to time, you may need to mannually test Harbor REST API. You can deploy the Swagger file into Harbor's service node. We suppose you install Harbor through online or offline installer, then you should have a Harbor directory after you untar the installer, such as **~/harbor**.
```

Comment:
Suppose you install Harbor through the online or offline installer, then you should have a Harbor directory after you un-tar the installer,